modules/popup/css/lib/popup.scss

```diff
@@ -1,203 +1,120 @@
-.wdc-popup {
-  @include wdc-type();
-  @include wdc-type-body();
-  @include wdc-depth-3();
-  display: inline-block;
-  background-color: $_wdc-popup-bg-color;
-  padding: $wdc-spacing-s;
-  border-radius: 3px;
-  border: 1px solid $_wdc-popup-border-color;
-  position: relative;
-  width: max-content;
-  max-width: $_wdc-popup-max-width;
-
-  // Fallback for IE11
-  @media screen and (-ms-high-contrast: active),
-  screen and (-ms-high-contrast: none) {
-    width: auto;
-    min-width: $_wdc-popup-menu-min-width;
+@keyframes popupAnimationTopCenter {
```

Comment:
This is the right approach, but we will also need motion for TopLeft, TopRight, BottomLeft, BottomRight. You could look at an approach like cards and grids to programmatically generate these

Reply:
Do you mind moving the animation stuff to another file as well?

Reply:
sure thing!